src/main/resources/swagger/api-docs.yaml

```diff
@@ -2976,6 +2985,44 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorReport'
+  /api/resources/v2/{resourceTypeName}/{resourceId}/creator:
```

Comment:
This is fine but it's probably more helpful to have a way to batch the lookup of `resource => createdBy`. Either here or in the `listResources` response perhaps?